downstream/modules/eda/proc-eda-set-up-credential-types.adoc

```diff
@@ -0,0 +1,169 @@
+[id="eda-set-up-new-credential-types"]
+
+= Creating a new credential type
+
+You can create a credential type to use with a source plugin or a private container registry that you select based on the supported, default credential types. You can make your credential type available to a team or individuals.
```

Comment:
@jself-sudoku You don't need a custom credential type for Container Registry we already have a Credential type for that. Credential Types should only be added for Source Plugins in EDA, there is currently no other use case for it. For anything that is used internally by EDA we already provide Credential types. Credential Types are available to everyone when a Credential Type is created you won't see a "Organization" field.

Reply:
Thank you for this information, @mkanoor.  Very useful.